### ISSUE TITLE
Improve Bind module link

### ISSUE THREAD

**Issue by shoulders** [webmin/webmin#2225]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.4     |
|  Webmin version      |     2.111      |
|  Usermin version     |     2.010     |
|  Virtualmin version  |     7.20.1       |
|  Theme version       |     21.10      |
|  Package updates     |     All installed packages are up to date   |

### Background

The links do what they should and are not broken.

This is a small issue but I thought I would mention it 😄 

### The issue

On the home page, the 2 highlighted links point to `https://devweb.svchost.uk:10000/bind8/` but we are using bind9

![image](https://github.com/user-attachments/assets/fa403651-cc78-4aee-adb2-8d8a98a65940)

![image](https://github.com/user-attachments/assets/1284a0b9-bc76-42dd-b1f7-fda9c1f4114d)

### proposed solution

Change the links to point to `bind9` or just `bind`

If this required the module to be renames, then maybe it should be renamed to `bind` to remove the number.

**Comment by jcameron** [comment]
Yeah the module is named "bind8" for historical reasons, because in the old old days there was a separate module for BIND 4. It's really hard to rename modules, so in practice we're not going to change it..
